docs/docsource/operation/XDM_GUI/file_operations/view_file_info.rst

```diff
@@ -0,0 +1,59 @@
+.. _View File Info:
+
+View File Info
+==============
+
+To view information about a file:
+
+#. Click the **Data Browser** tab open the Data Browser view.
+
+   .. image:: ../../Graphics/xdm_ui_data_browser.png
+      :width: 75%	      
+
+#. Select a bucket by clicking its name.
+
+   .. image:: ../../Graphics/xdm_ui_bucket_select.png
+      :width: 75%
+
+#. Select the file by clicking in its row.
+
+   .. image:: ../../Graphics/xdm_ui_file_select.png
+      :width: 75%
+
+   .. note::
+
+      Clicking the filename downloads the file.
+
+#. Summary data about the file is displayed in the right column.
+
+   .. image:: ../../Graphics/xdm_ui_file_info_summary.png
+      :width: 50%
+
+#. To review or modify the file's metadata, click the **Metadata** tab.
+
+   .. image:: ../../Graphics/xdm_ui_file_info_metadata.png
+      :width: 50%
```

Comment:
Maybe make this image bigger for a better accessibility?

Reply:
Not a bad plan.